service-api/module/Application/config/module.config.php

```diff
@@ -399,6 +401,10 @@
                 $tableName,
                 $serviceLocator->get(LoggerInterface::class)
             ),
+            SessionStatusService::class => fn(ServiceLocatorInterface $serviceLocator) => new SessionStatusService(
+                $serviceLocator->get(YotiServiceInterface::class),
+                $serviceLocator->get(DataImportHandler::class)
+            ),
```

Comment:
ah yes, the auto-wiring did not seem to work here without with the following error: `{
    "status": 500,
    "type": "UnexpectedError",
    "title": "An unexpected error occurred",
    "detail": "Unable to create controller \"Application\\Controller\\YotiController\"; unable to resolve parameter \"sessionService\" using type hint \"Application\\Yoti\\SessionStatusService\"",
    "exception": "Laminas\\ServiceManager\\Exception\\ServiceNotFoundException"
}`

Alternatively, could add this as an invokable factory explicitly with `SessionStatusService::class => InvokableFactory::class`

but that requires creating a factory class for SessionStatus that doesnt have much else in way to configure?

Reply:
Sorry I've since spotted that laminas-di has never been installed so there's no autowiring. The way it is is fine for now, though we should probably add autowiring at some point